### PR TITLE
Added Docker for API services

### PR DESCRIPTION
**Pull request by kaleanych**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -281,4 +281,21 @@ refresh-mapping-files:
...
+
+#-----------------------------------------------------------------------------
+# API Service commands
+#-----------------------------------------------------------------------------
+build-api:
+	@ echo -e "$(BUILD_PRINT) Build API services $(END_BUILD_PRINT)"
+	@ docker-compose -p ${ENVIRONMENT} --file infra/api/docker-compose.yml --env-file ${ENV_FILE} build --no-cache --force-rm
+	@ docker-compose -p ${ENVIRONMENT} --file infra/api/docker-compose.yml --env-file ${ENV_FILE} up -d --force-recreate
+
+start-api:
```
Comment (costezki) [inline Makefile:295]: call it by its name, pls: `identity_manager`, or whatever was the component.

File: infra/api/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+version: '3'
+
+services:
+  api-id_manager:
```
Comment (costezki) [inline infra/api/docker-compose.yml:4]: use a precise name pls.

Comment (costezki) [reply infra/api/docker-compose.yml:4]: ID_MANAGER_API

File: Makefile (hunk 1 above)
Comment (costezki) [reply Makefile:295]: ID_MANAGER_API

File: infra/api/id_manager/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,11 @@
+FROM python:3.8-slim-buster
+EXPOSE $ID_MANAGER_API_PORT
+
+WORKDIR /usr/src/api/id_manager
+COPY . /usr/src/api/id_manager
+
+RUN pip install --upgrade pip
+RUN pip install -r requirements.txt
+RUN pip install -r infra/api/requirements.txt
```
Comment (costezki) [inline infra/api/id_manager/Dockerfile:9]: please move "infra/api/requirements.txt" -> requirements-infra.txt

File: ted_sws/notice_validator/entrypoints/cli/cmd_shacl_runner.py [hunk 4]
```diff
@@ -13,7 +13,7 @@
 
 DEFAULT_RDF_FOLDER = '{mappings_path}/{mapping_suite_id}/' + DEFAULT_OUTPUT_PATH
 DEFAULT_TEST_SUITE_REPORT_FOLDER = "test_suite_report"
-HTML_REPORT = "{id}.html"
+HTML_REPORT = "shacl_{id}.html"
```
Comment (costezki) [inline ted_sws/notice_validator/entrypoints/cli/cmd_shacl_runner.py:16]: nice!

File: ted_sws/notice_validator/entrypoints/cli/cmd_sparql_runner.py [hunk 5]
```diff
@@ -13,8 +13,8 @@
 
 DEFAULT_RDF_FOLDER = '{mappings_path}/{mapping_suite_id}/' + DEFAULT_OUTPUT_PATH
 DEFAULT_TEST_SUITE_REPORT_FOLDER = "test_suite_report"
-JSON_REPORT = "{id}.json"
-HTML_REPORT = "{id}.html"
+JSON_REPORT = "sparql_{id}.json"
+HTML_REPORT = "sparql_{id}.html"
```
Comment (costezki) [inline ted_sws/notice_validator/entrypoints/cli/cmd_sparql_runner.py:17]: nice

**Review by costezki — APPROVED** [review]
Looking good! 
pls rename a few things

File: infra/api/docker-compose.yml (hunk 2 above)
Comment (kaleanych) [reply infra/api/docker-compose.yml:4]: semnatic, it's the same thing :) I've put "api" as prefix, so all api containers will be neighbours in the list.
Prefix is the functionlity of container (in this case "api") and the last part is the name of service (id_manager)

File: infra/api/id_manager/Dockerfile (hunk 3 above)
Comment (kaleanych) [reply infra/api/id_manager/Dockerfile:9]: why?
it's strictly related to APIs containers.
why install it in every container, as it's context-based?

File: Makefile (hunk 1 above)
Comment (kaleanych) [reply Makefile:295]: this target is meant to start all APIs (it may be others in the future)

**Pull request merged**